code-challenges.js

```diff
@@ -17,58 +17,94 @@
+const myNumbers1 = [3, 7, 0, 36, -9]
+const myNumbers2 = [8, -7, 42, 9, 13]
 
-//Use an accessor to combine two arrays.
-//Use length to return the length.
+// Pseudocode
+// Create a function name comboArrays.
+// Use the concat method to combine two arrays.
+// Use length to return the number of elements.
+// Expected output: 10
+
+var comboArrays = myNumbers1.concat(myNumbers2)
+console.log(comboArrays.length)
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ovarB"
 
 const currentCohort = "Bravo 2022"
 
-console.log(currentCohort.reverse())
+// Pseudocode
+// Create a function named reverseString.
+// Use a split() method to return a new array.
+// Use the reverse() method to reverse the new array.
+// Use the join() method to join all the elements of the array.
+// Return the reversed string.
+// Expected output: 2202 ovarB
+
+function reverseString(str) {
+    var splitString = str.split("")
+    var reverseArray = splitString.reverse()
+    var joinArray = reverseArray.join("")
+    return joinArray
+}
+console.log(reverseString(currentCohort))
 
 //--------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
-// const myArray = [13, 34, 5, 10, 27, 42]
-//
-// for(let i = 0; i < myArray.length; i ++) {
-//   if (myArray [i] % 2 == 0) {
-//     console.log(`${myArray[i]} is an even number`)
-//   } else { console.log(`${myArray[i]} is an odd number`)
-//   }
-// }
+const myArray = [13, 34, 5, 10, 27, 42]
+
+// Pseudocode
+// Create a conditional statement
+// Access each item in the array using iteration with a for loop
+// Evaluate whether each number in array is even or odd
+// Expected output: odd, even, odd, even, odd, even
+
+for(let i = 0; i < myArray.length; i ++) {
+  if (myArray [i] % 2 == 0) {
+    console.log("even")
+  } else {
+    console.log("odd")
+  }
+}
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
-//  Set one:
-// const number1 = 58
-// const number2 = 100
-//
-//  Set two:
-// const number3 = 27
-// const number4 = 24
-//
-// const newNumber = () => {
-//   return (number2 - number1)
-// }
-// console.log(newNumber())
+// Set one:
+const number1 = 58
+const number2 = 100
+
+// Set two:
+const number3 = 27
+const number4 = 24
+
+// Pseudocode
+// Create an if function.
+// Use a conditional statement to take in two numbers and subtracts the smaller number from the large.
+// Expected output: 42, 3
+
+  if(number1 < number2 && number3 < number4){
+    console.log(number1 - number2, (number3 - number4))
+  } else {
+    console.log(number2 - number1, (number3 - number4))
+  }
```

Comment:
You had the logic with your first commit on this problem.  To make it more dynamic, the function would look something like this: `const newNumber = (num1, num2) => {
  return (num2 - num1)
}`